modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/footer/index.html

```diff
@@ -0,0 +1,17 @@
+<div class="cp_footer_home">
+<div class="cp_footer_container">
+	<h6 data-lfr-editable-id="cp_footer_text" data-lfr-editable-type="rich-text" class="cp_footer_text">
+	  © 2023 Liferay Inc. All Rights Reserved &nbsp; &nbsp; &nbsp; &nbsp; |
```

Comment:
can you indent this line with a full tab instead of 2 spaces

Reply:
Done!